nameresolution/kubernetes/kubernetes.go

```diff
@@ -45,21 +58,34 @@ func (k *resolver) Init(metadata nameresolution.Metadata) error {
 	if err != nil {
 		return err
 	}
-	if config, ok := configInterface.(map[string]interface{}); ok {
-		clusterDomainPtr := config[ClusterDomainKey]
+	if cfg, ok := configInterface.(map[string]interface{}); ok {
+		clusterDomainPtr := cfg[ClusterDomainKey]
```

Comment:
I know the name "ptr" was there before, but is it possible to rename it to something else? Since technically it's not a pointer, just an interface. Maybe "clusterDomainAny" or anything else? Same for "tmplStr"